fmriprep/cli/run.py

```diff
@@ -222,6 +222,9 @@ def get_parser():
         '--fs-license-file', metavar='PATH', type=Path,
         help='Path to FreeSurfer license key file. Get it (for free) by registering'
              ' at https://surfer.nmr.mgh.harvard.edu/registration.html')
+    g_fs.add_argument(
+        '--coreg-init-header', action='store_true', dest='coreg_init_header', default=False,
+        help='Use header information to initialize volume corergistration')
```

Comment:
This should not be a FreeSurfer-specific option, as we also have an FSL implementation that should behave the same.

I would suggest something like `--bold2t1w-init` and take the options "register" and "header", defaulting to "register". This pairs it naturally with `--bold2t1w-dof`, IMO.

Reply:
So I'm not familiar with FSL or its way of doing things--do you know how an option like `--bold2t1w-init=header` gets passed to FLIRT? I have made a stab at adding it (using the `apply_xfm` input option, but this isn't something I use and is probably not correct.